src/status_im/core.cljs

```diff
@@ -9,7 +9,7 @@
             cljs.core.specs.alpha))
 
 (if js/goog.DEBUG
-  (.ignoreWarnings (.-YellowBox js-dependencies/react-native)
+  (.ignoreWarnings (    .-YellowBox js-dependencies/react-native)
```

Comment:
?

Reply:
Look at the commit title: `break linting for sake of test`
I'm showing that it works, look at this build:
https://ci.status.im/blue/organizations/jenkins/status-react%2Fprs%2Fandroid/detail/PR-10224/2/pipeline
The `Lint` stage failed, but the build continued and uploaded the APK.

Reply:
Now I will remove that commit.